src/Pages/jihyunglee/Main/components/Search.js

```diff
@@ -1,90 +1,33 @@
-          id: "wedding_limwegi",
-          nickname: "이예림 플래너 | 베리굿웨딩",
-          picture:
-            "https://scontent-gmp1-1.cdninstagram.com/v/t51.2885-19/s150x150/100767610_1180085765716998_1803428214647816192_n.jpg?_nc_ht=scontent-gmp1-1.cdninstagram.com&_nc_ohc=2TkZdzw15bcAX95u-8X&oh=e05f8e7959bb2743170a8aeb6a8936ce&oe=5F6FC96E",
-        },
-        {
-          id: "wecode_founder",
-          nickname: "송은우 (Eun Woo Song)",
-          picture:
-            "https://scontent-gmp1-1.cdninstagram.com/v/t51.2885-19/s150x150/95487199_185290135875370_3869726896170205184_n.jpg?_nc_ht=scontent-gmp1-1.cdninstagram.com&_nc_ohc=Mc2n37ZUt-YAX_uuSnU&oh=da46ba94f0e7f252813118343352f5f8&oe=5F7240C5",
-        },
-        {
-          id: "shall_we_moncher",
-          nickname: "몽쉘",
-          picture:
-            "https://scontent-gmp1-1.cdninstagram.com/v/t51.2885-19/s150x150/73361719_819771735159714_3093402774210084864_n.jpg?_nc_ht=scontent-gmp1-1.cdninstagram.com&_nc_ohc=_27Mx444a3wAX-Rlp84&oh=540e6d0b29d2382c6edfd3398f8fa8e9&oe=5F70D4BF",
-        },
-        {
-          id: "welcometosoomin",
-          nickname: "SOOMIN",
-          picture:
-            "https://scontent-gmp1-1.cdninstagram.com/v/t51.2885-19/s150x150/80604651_569390977037188_1043545901806649344_n.jpg?_nc_ht=scontent-gmp1-1.cdninstagram.com&_nc_ohc=l_AyCjOv94UAX9PhYlW&oh=53e4f29aef04e2404af997b4710ed49e&oe=5F702588",
-        },
-        {
-          id: "wecode_korea",
-          nickname: "",
-          picture:
-            "https://instagram.fcnq2-2.fna.fbcdn.net/v/t51.2885-19/44884218_345707102882519_2446069589734326272_n.jpg?_nc_ht=instagram.fcnq2-2.fna.fbcdn.net&_nc_ohc=7xEzH-b7neEAX8c6Zss&oh=13cd686873745bec4aa9aea6ade7d2d5&oe=5F6EA40F&ig_cache_key=YW5vbnltb3VzX3Byb2ZpbGVfcGlj.2",
-        },
-        {
-          id: "diuun_",
-          nickname: "Jiyoon",
-          picture:
-            "https://scontent-gmp1-1.cdninstagram.com/v/t51.2885-19/s150x150/92815808_654640765087774_3569504833889107968_n.jpg?_nc_ht=scontent-gmp1-1.cdninstagram.com&_nc_ohc=IWig0tbhq5IAX9pYuIX&oh=ea8c41b47956e02c59b656ea6f21b467&oe=5F6F5643",
-        },
-        {
-          id: "ssem_ni",
-          nickname: "세 민",
-          picture:
-            "https://scontent-gmp1-1.cdninstagram.com/v/t51.2885-19/s150x150/117155857_903863120099240_2371507172497494303_n.jpg?_nc_ht=scontent-gmp1-1.cdninstagram.com&_nc_ohc=MG5LtHwXexUAX8JmCGN&oh=17c235482e73a66bfe99d6173930e198&oe=5F7205F2",
-        },
-      ],
+      userArr: [],
       matchArr: [],
       focusIn: false,
     };
   }
 
+  componentDidMount() {
+    this.setState({
+      userArr: SEARCH,
+    });
+  }
+
   checkMatch = (e) => {
     const searchValue = e.target.value;
     const { userArr } = this.state;
     this.setState({
-      matchArr: userArr.filter((el) => el.id.includes(searchValue)),
+      matchArr:
+        searchValue !== ""
+          ? userArr.filter((el) => el.id.includes(searchValue))
+          : [],
```

Comment:
truthy, falsy value에 대해서 다시 한 번 정리해주세요!

```suggestion
      matchArr:
        searchValue
          ? userArr.filter((el) => el.id.includes(searchValue))
          : [],
```